org.jupyter.JupyterLab.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: org.jupyter.JupyterLab
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
+rename-desktop-file: jupyterlab-desktop.desktop
+rename-icon: jupyterlab-desktop
+modules:
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2023.02beta.1
+        commit: 64f6ccfdfd223cc2d2d68608ef42b8ff2fd6400e
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+(?:beta[\d.]+?))$
+  - name: jupyterlab
+    buildsystem: simple
+    build-commands:
+      - ar -x JupyterLab-Setup-Debian.deb
+      - rm -f JupyterLab-Setup-Debian.deb
+      - tar -xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r opt/* usr/* $FLATPAK_DEST
+      - rm -rf opt usr
+      - install -Dm 755 run.sh $FLATPAK_DEST/bin/
+      - desktop-file-edit --set-key=Exec --set-value="run.sh %U" $FLATPAK_DEST/share/applications/jupyterlab-desktop.desktop
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/jupyterlab/jupyterlab-desktop/releases/download/v3.6.1-2/JupyterLab-Setup-Debian.deb
```

Comment:
You still build a aarch64 flatpak.

Reply:
Yes, I saw. I've added a `flathub.json` .